releases/release-1.22/release-notes/release-notes-draft.json

```diff
@@ -3405,8 +3405,8 @@
   },
   "101977": {
     "commit": "2a106464d355d04f1c8931a883d265cf70612a30",
-    "text": "kubeadm: fix the bug that kubeadm only uses the first hash in `caCertHashes` to verify the root CA",
-    "markdown": "Kubeadm: fix the bug that kubeadm only uses the first hash in `caCertHashes` to verify the root CA ([#101977](https://github.com/kubernetes/kubernetes/pull/101977), [@SataQiu](https://github.com/SataQiu))",
+    "text": "kubeadm: fix the bug that `kubeadm` only uses the first hash in `caCertHashes` to verify the root CA.",
+    "markdown": "Kubeadm: fix the bug that `kubeadm` only uses the first hash in `caCertHashes` to verify the root CA. ([#101977](https://github.com/kubernetes/kubernetes/pull/101977), [@SataQiu](https://github.com/SataQiu))",
```

Comment:
@pmmalinov01 already took care of this one via a seperate PR: https://github.com/kubernetes/sig-release/pull/1637

Reply:
Yep. The pr you mentioned fixed the yaml file so the draft json file here could catch the right text. Thanks

Reply:
Yup, you are right. I gave it a lgtm. Good catch!!

Reply:
It is fine, krel flow update the markdown files :)